doc/lua_api.md

```diff
@@ -7602,98 +7603,98 @@ Misc.
 Global objects
 --------------
 
-* `minetest.env`: `EnvRef` of the server environment and world.
-    * Any function in the minetest namespace can be called using the syntax
-      `minetest.env:somefunction(somearguments)`
-      instead of `minetest.somefunction(somearguments)`
+* `core.env`: `EnvRef` of the server environment and world.
+    * Any function in the core namespace can be called using the syntax
+      `core.env:somefunction(somearguments)`
+      instead of `core.somefunction(somearguments)`
     * Deprecated, but support is not to be dropped soon
```

Comment:
```suggestion
* `core.env`: `EnvRef` of the server environment and world.
    * Any function in the `core` namespace can be called using the syntax
      `core.env:somefunction(somearguments)`
      instead of `core.somefunction(somearguments)`
    * Deprecated, but support is not to be dropped soon
* `minetest`: alias for the `core` namespace
    * Deprecated, but support is not to be dropped soon
```